docs/learn/learn-xcm-pallet.md

```diff
@@ -10,12 +10,90 @@ slug: ../learn-xcm-pallet
 The XCM pallet ([`pallet-xcm`](https://github.com/paritytech/polkadot/tree/master/xcm/pallet-xcm))
 provides a developer-friendly interface for most of the common XCM messages.
 
-This pallet provides default implementations for many traits required by `XcmConfig`, which is
-highly useful for configuration.
+This pallet provides some default implementations for some traits required by `XcmConfig`, as well
+as an instance of the XCM Executor provided as a trait within the pallet's own configuration.
```

Comment:
If it's provided as a trait, then it's not providing an instance, because by its very nature, traits are abstract concepts and not concrete implementations.